pyaerocom/aeroval/helpers.py

```diff
@@ -181,10 +184,30 @@ def reorder_experiments_menu_evaluation_iface(menu_file, exp_order=None):
     new = sort_dict_by_name(current, pref_list=exp_order)
     write_json(new, menu_file, indent=4)
 
-def _check_statistics_periods(periods):
+def _check_statistics_periods(periods: list) -> list:
```

Comment:
Shouldn't list annotations have type of list elements annotated? So list[str]?

Reply:
Good point, sounds reasonable (I am also newish to the world of type annotations).

Reply:
Same, only knew about it because someone commented on one of my PRs a couple of weeks ago :P Other than that, the PR looks good